res/controllers/Midi_for_light.midi.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MixxxMIDIPreset mixxxVersion="1.12+" schemaVersion="1">
+	<info>
+		<name>MIDI for light</name>
+		<author>Michael Stahl / DG3NEC</author>
+		<description>MIDI-output for light control</description>
+		<forums>http://www.mixxx.org/forums</forums>
```

Comment:
Could you create a stub forum post to link to here? (just post in the controllers section with an empty post with the title of your preset)